tests/Responses/ajax/product/index_response.json

```diff
@@ -0,0 +1,160 @@
+{
+    "page": 1,
+    "limit": 10,
+    "pages": 3,
+    "total": 22,
+    "_links": {
+        "self": {
+            "href": "\/admin\/ajax\/products\/?page=1&limit=10"
+        },
+        "first": {
+            "href": "\/admin\/ajax\/products\/?page=1&limit=10"
+        },
+        "last": {
+            "href": "\/admin\/ajax\/products\/?page=3&limit=10"
+        },
+        "next": {
+            "href": "\/admin\/ajax\/products\/?page=2&limit=10"
+        }
```

Comment:
The only differences with `tests/Responses/product/index_response.json` are those lines, I don't know if there is a way to avoid creating two nearly same files.